src/components/templates/ServiceBanner.js

```diff
@@ -0,0 +1,85 @@
+import React, { Component } from 'react';
+
+
+
+class ServiceBanner extends Component {
+  
+
+
+  handleClick = e => {
+    e.preventDefault()
+    console.log(e.target.innerHTML) 
+
+    const ul = document.getElementsByClassName('services-ul')
```

Comment:
With React, we try not to target the DOM directly; use state and props to update the elements we're trying to target.